swan-lake/learn-the-platform/native-support/build-a-native-executable.md

```diff
@@ -323,26 +323,19 @@ After the environment is set up, follow the steps below to build the native exec
    ```
 
    The Docker file :
-   ```	
+   ```dockerfile	
```

Comment:
dockerfile is not a language type.  Select the appropriate language from https://github.com/shikijs/shiki/blob/main/docs/languages.md

Reply:
Will change this once the branch is sync with master

Reply:
Removed `docker` and will add after master sync